src/App.js

```diff
@@ -1,25 +1,45 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import Movies from './Movies';
+import SingleMovie from './SingleMovie';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      singleMovie: false,
+      error: '',
+    }
+  };
+
+  componentDidMount = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Feel free to move this fetch call into it's own file and import it in for use just like y'all used to do in Mod 2.